### PR TITLE
add dark overlay variant for img/text collision

### PR DESCRIPTION
**Pull request by kenworthyc**
update second city next-button image, add dark overlay so text is readable

### CODE REVIEW

File: src/templates/partials/work-next-button.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<button class="work-next-page{{#if next.variant}} work-next-page--{{ next.variant }}{{/if}} {{ next.class }}"
+<button class="work-next-page{{#each next.variant}} work-next-page--{{ this }}{{/each}} {{ next.class }}"
   ng-click="app.nextButton()"
```
Comment (krispy1298) [inline src/templates/partials/work-next-button.html]: @kenworthyc does this affect pages with 0 or 1 variant negatively?

Comment (kenworthyc) [inline src/templates/partials/work-next-button.html]: worked for 0 variants, but had to restructure the yamls for 1 variant. thankssss

Comment (krispy1298) [inline src/templates/partials/work-next-button.html]: maybe a if and each are needed?

Comment (krispy1298) [inline src/templates/partials/work-next-button.html]: maybe at this point, it's more semantic to say `variation` instead?

Comment (kenworthyc) [inline src/templates/partials/work-next-button.html]: sure, that makes sense

**Pull request merged**